crates/release_plz_core/src/command/release.rs

```diff
@@ -303,6 +314,22 @@ impl GitReleaseConfig {
         self.draft = draft;
         self
     }
+
+    pub fn set_release_type(mut self, release_type: ReleaseType) -> Self {
+        self.release_type = release_type;
+        self
+    }
+
+    pub fn is_pre_release(&self, git_tag: &str) -> bool {
+        match self.release_type {
+            ReleaseType::Pre => true,
+            ReleaseType::Auto => match Version::parse(git_tag.trim_start_matches('v')) {
```

Comment:
git tags can also start with the package name. E.g. [release-plz-v0.3.41](https://github.com/MarcoIeni/release-plz/releases/tag/release-plz-v0.3.41).
Can you use `package.version` instead? You have it [here](https://github.com/ybaldus/release-plz/blob/97d73d7c409bd15279bdf5ede73c49c72f140ae5/crates/release_plz_core/src/command/release.rs#L505)